src/views/AddPriceSingle.vue

```diff
@@ -430,7 +447,11 @@ export default {
         this.setLocationData(this.recentLocations[0])
       }
       this.addPriceSingleForm.price_per = this.categoryPricePerList[0].key // init to 'KILOGRAM' because it's the most common use-case
-      this.addPriceSingleForm.currency = this.appStore.user.last_currency_used
+      this.addPriceSingleForm.currency = this.appStore.getUserLastCurrencyUsed
+      this.isChangeCurrency = false
+    },
+    goToSettings() {
```

Comment:
I'm not sure this is a good idea to put a link to the settings here. the form will be lost, the user doesn't know that. Alternatives :
- clear message that form will be lost
- allow changing the currency settings in a popup (without going to the settings)